tests/create_appliance.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3
+
+from swsscommon import swsscommon
+from dash_api.appliance_pb2 import *
+import typing
+import ipaddress
+import socket
+import sys
+
+def to_string(value):
+    if isinstance(value, bool):
+        return "true" if value else "false"
+    elif isinstance(value, bytes):
+        return value
+    return str(value)
+
+class ZmqProduceStateTable(object):
```

Comment:
>ZmqProduceStateTable [](http://example.com/codeflow?start=6&length=20)

The class name is quite confusing. Why we need it?

Reply:
Fixed, removed this class.